packages/components/src/Chip/Chip.module.css

```diff
@@ -107,6 +107,7 @@
 .subtle.invalid {
   --chip-border-color: var(--color-critical);
   --chip-base-bg-color: var(--color-critical--surface);
+  --chip-base-hover-bg-color: var(--color-interactive--background--hover);
```

Comment:
since we're not trying to change the value, rather now we have 2 separate gradients that we are swapping between - it no longer makes sense to have this defined only in the hover state rules. we know it should always be this color.